### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by eamador1**
Since the last review, the following changes were made in the project:

- [ ] Create and style desktop version
- [ ] Run linters

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -576,3 +569,437 @@ footer {
...
+  }
+
+  .container-touch {
+    justify-content: left;
+  }
+
+  .contactme .contactform .division {
+    width: 100vw;
+    margin-left: 0;
+  }
```
Comment (frempongdev) [inline style.css:990]: Great job so far with your desktop version. 👍🏿 There is one tiny change however I would suggest you make. ♻️ 
Currently, your page extends beyond the normal width of the screens. 
divider too long

- [ ] I suggest that you check the width above and adjust it to fit the page. ↔️

File: style.css [hunk 2]
```diff
@@ -576,3 +569,437 @@ footer {
...
+
+  .hey {
+    font-size: 72px;
+    line-height: 81.5px;
+  }
+
+  .paragraph {
+    font-size: 20px;
+    line-height: 32px;
+  }
```
Comment (frempongdev) [inline style.css:651]: Great going so far 🔥. We have some tiny `font` requirements to meet here 🤝 

- [ ] Kindly make sure you stick to the Figma template requirements as closely as possible. The `font family` for the Desktop version is `Inter`. Please make sure you check all the other differences in fonts and fix them. ♻️ 

image

**Review by frempongdev — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @eamador1   👋🏼 
I'm Frempong.

You've done a great job implementing most of the requirements for this project 🚀, but there are a few changes I recommend be made. You're almost done, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ No Linter errors. 🆗  
✅ UI design

 ## Required Changes ♻️
_Kindly Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eamador1** [comment]
Since the last review, the following changes were made in the project:

- [ ] Change ".contactme .contactform .division" width from 100vw to 100% to avoid horizontal scroll
- [ ] Establish "footer .division" margins so that division line occupies 100 vw
- [ ] Run linters

**Review by roshan-bajgain — APPROVED** [review]
Hi @eamador1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Good job on implementing the changes ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**